src/tests/conftest.py

```diff
@@ -495,6 +495,33 @@ def movie(category) -> Movie:
     return result
 
 
+@pytest.fixture()
+def movies_dataset() -> dict:
+    j = json.loads((HERE / "files" / "movies.json").read_text())
+    s = DatasetSchema.from_dict(j)
+    return Dataset.create_for_schema(s)
+
+
+@pytest.fixture
+def movies_category(movies_dataset, dynamic_models):
```

Comment:
You could add a return type here too, for consistency.

Reply:
I don't really know what the type should be...

Reply:
Jan, het enige mogelijke returntype is `Type[DynamicModel]` want het echte model wordt in-memory gemaakt. Ik denk dat dit niet heel veel toevoegd voor de fixture (al linkt PyCharm dat wel)